src/wx/autoupdater/wxmsw/autoupdater.cpp

```diff
@@ -0,0 +1,27 @@
+#include "../autoupdater.h"
+#include "../../../common/version_cpp.h"
+#include "winsparkle-wrapper.h"
+
+
+void initAutoupdater()
+{
+    wxString version(vbam_version);
+    size_t index = version.Find("-");
```

Comment:
Maybe you can use my contains template function in src/common/contains.h.

I think we can drop support for non-stl wx builds.

Reply:
Also thank you for fixing this. This will really help with the nightlies a lot. I need to do some more work on the windows server too.

Reply:
Sorry, I misread the code, you are using the index to trim the value.

I'd write it like this:

```cpp
if ((size_t index = version.Find('-')) != wxString::npos)
    version = version.SubString(0, index - 1);
```

Reply:
Also I have a `str_split()` function in strutils.cpp, which you could use like this:

```cpp
wxString version = str_split(vbam_version, '-')[0];
```

untested, but should work.